input/templates/partials/catalog/faq-accordion.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="panel panel-default faq-accordion-panel">
+  <div class="panel-heading" role="tab" id="accordion-{{index}}">
+    <h4 class="panel-title faq-accordion-title">
+      <a id="pdp-product-details-toggle" class="collapsed pdp-accord-toggle" data-toggle="collapse" data-parent="#accordion-product-info" href="#accordion-collapse-{{index}}" aria-expanded="false" aria-controls="accordion-collapse-{{index}}">{{title}}
```

Comment:
A little change: `{{index}}` that I've used in the templates is actually coming from Handlebars:
http://handlebarsjs.com/reference.html#data-index

So to fix this, simply change to `{{@index}}` and in the JSON remove the `index` field.